program.js

```diff
@@ -0,0 +1,63 @@
+var buttons = document.querySelector("#submi");
+buttons.addEventListener('click', changesome);
+
+function changesome() {
+    var button1 = document.querySelector('#inputname');
+    var button1value = button1.value;
+    if (button1value.length == 0) {
+        alert("please enter a Name");
+        return
+    }
+    var button2 = document.querySelector('#inputEmail');
+    var button2value = button2.value;
+    if (button2value.length == 0) {
+        alert("please enter a email");
+        return
+    }
+    var emailaddres = document.querySelector('#inputEmail');
+    if (emailaddres.value.indexOf('@') === -1) {
```

Comment:
You could improve your validation a bit further by validating that the E-mail address does not begin or end with '@' and only contains one '@' symbol. For example:

@gordon@.blah.com would not be a valid E-mail address.

You could also make sure that the E-mail address does not contain any symbols (like ^%&*!"().) hi£$"%@things.com would not be valid, for example.

And you could make sure that there is a '.' somewhere after the '@'.

Note: it's difficult to validate E-mail addresses properly, so we would probably use a library to do it for us. But it would be good practise for you to try those things above :).